runbooks/how-to-add-or-remove-user-SSH-access.md

```diff
@@ -0,0 +1,442 @@
+
+19. Find the block of SSM keys corresponding to SSH, or, if the block of keys do not yet exist,
+    append the below to the end of the file. `....` should be replaced with the user's EUA ID. What
+    each key represents is as follows:
+
+    1. `ssh_access` defines whether the user has the ability to SSH into instances launched in the
+       current environment. If this is `false`, the user will not have the ability to SSH into any
+       instance, whereas if this is `true` the user _will_ be able to SSH. Note that this is
+       overriden by `ssh_default_access` (in the `mgmt` "environment") if `ssh_default_access` is
+       `true`. Omission is treated as `false`
+    2. `ssh_sudoer` defines whether the user has the ability to run `sudo` and act as `root` on
+       instances launched in the current environment. If this is `false`, the user will not have the
+       ability to run `sudo` and act as `root`, whereas if this is `true` the user _will_ be able to
+       do so. Note that this is overriden by `ssh_default_sudoer` (in the `mgmt` "environment") if
+       `ssh_default_sudoer` is `true`. Omission is treated as `false`
+
+    ```yaml
+    /bfd/${env}/common/sensitive/user/..../ssh_access: bool
+    /bfd/${env}/common/sensitive/user/..../ssh_sudoer: bool
+    ```
+
+20. Close the file. This will save your changes and re-encrypt the encrypted YAML file for the
+    current environment.
+21. Plan the changes to the Terraform state and verify that there are only _additions_ to the state
+    and that these additions correspond to the new SSH SSM parameters defined in step #20:
+
+    ```bash
+    terraform plan
+    ```
+
+22. Repeat steps #14 through #21 for each environment that the user should be provisioned in
+23. Open a new Pull Request with the changes to all configuration in the associated branch
+24. Once approved, the changes to `mgmt` can be applied:
+
+    1. From the root of the repository, `cd` into the `mgmt` module:
+
+       ```bash
+       cd ops/terraform/env/mgmt
+       ```
+
+    2. Apply the changes to configuration ensuring that there are no unexpected changes:
+
+       ```bash
+       terraform apply
+       ```
+
+25. Once approved, the changes to each established environment can also be applied:
+
+    1. `cd` into the `base` service Terraform module:
+
+       ```bash
+       cd ops/terraform/services/base
+       ```
+
+    2. Switch to the current environment's corresponding workspace:
+
+       ```bash
+       terraform workspace select <prod|prod-sbx|test>
+       ```
+
+    3. Apply the changes to configuration ensuring that there are no unexpected changes:
+
+       ```bash
+       terraform apply
+       ```
+
+    4. Repeat the above steps for each established environment that was changed
+
+26. Once finished, the new user will be _fully_ provisioned the next time the `Build App AMIs` stage
+    is ran in our deployment pipeline. In the meantime, anytime a _brand new_ instance is _launched_
```

Comment:
```suggestion
    is run in our deployment pipeline. In the meantime, anytime a _brand new_ instance is _launched_
```
Profuse apologizes for my pedantry with past participles (and adjunct alliteration)